meta-mbl-distro/scripts/lib/mbl/update/payloadbuilder.py

```diff
@@ -35,6 +35,14 @@ class PayloadBuilder:
             ri.MBL_ROOTFS_ID: ri.RootfsImageV3,
             ai.MBL_APPS_ID: ai.AppsImageV3,
         },
+        4: {
+            pa.MBL_PAYLOAD_ARCHIVER_ID: pa.PayloadArchiverV3,
+            wi.MBL_WKS_BOOTLOADER1_ID: wi.WksBootloader1ImageV3,
+            wi.MBL_WKS_BOOTLOADER2_ID: wi.WksBootloader2ImageV3,
+            bi.MBL_BOOT_ID: bi.BootImageV3,
+            ri.MBL_ROOTFS_ID: ri.RootfsImageV4,
```

Comment:
Are we introducing any backward incompatibility with this new version?

Reply:
Depends what you mean. The new version of the update scripts won't be able to install old versions of payloads, but the payload script will still be able to create payloads using the old format version so you'll be able to update from 0.10 to 0.11 with a payload created in a 0.11 workarea by using `--mbl-os-version 0.10`.

If what you're thinking is "Do we need to update the *Version specific upgrade notes* section in the docs?" then the answer is "Yes".

Reply:
I suppose another question is "Should we support updating directly from 0.9 to 0.11?". Technically, we might be able to make that work using the same flow as 0.9 -> 0.10 updates, but TBH, I'm not convinced it's worth it. I suggest that except for 0.9 -> 0.10, we don't support upgrading from 0.9 because 0.9 doesn't support multi-component updates. Going from e.g. 0.10 -> 0.12 directly should be simpler.

Reply:
> I suppose another question is "Should we support updating directly from 0.9 to 0.11?". Technically, we might be able to make that work using the same flow as 0.9 -> 0.10 updates, but TBH, I'm not convinced it's worth it. I suggest that except for 0.9 -> 0.10, we don't support upgrading from 0.9 because 0.9 doesn't support multi-component updates. Going from e.g. 0.10 -> 0.12 directly should be simpler.

I don't think we should support 0.9 -> 0.11, because if we do then we will have to do from all future versions.
As soon as we make clear the _special_ step about updating from 0.9, I'm happy with it.

Reply:
Okay. There are also other related docs edits I need to do due to the rootfs label changes (including changes to the create-partition-table-markdown script).

Reply:
And the part of how to check if the rootfs was updated.